app/integration/runtime/src/main/java/io/syndesis/integration/runtime/logging/IntegrationActivityTrackingPolicyFactory.java

```diff
@@ -42,6 +42,6 @@ public boolean isInstance(RoutePolicy routePolicy) {
 
     @Override
     public boolean appliesTo(Flow flow) {
-        return flow.isPrimary();
+        return flow.isPrimary() || flow.isApiProvider();
```

Comment:
Perhaps we should define the exception rather than add refinements to the condition, something like:

```suggestion
        return !flow.isAlternate();
```

Reply:
@christophd just nitpicking, looks good, feel free to merge if there's no need to change this...